src/smt/smt_engine.cpp

```diff
@@ -946,6 +964,18 @@ class SmtEnginePrivate : public NodeManagerListener {
   std::ostream* getReplayLog() const {
     return d_managedReplayLog.getReplayLog();
   }
+  
+  //------------------------------- expression names
+  // implments setExpressionName, as described in smt_engine.h
```

Comment:
I'm following Clark's suggestion on this (from https://github.com/CVC4/CVC4/pull/1117):

barrettcw 12 days ago Owner
For these functions, just flag that they are the implementation versions of the functions defined in the header so you don't have to repeat the method comments.

Is there a part of the style guide that addresses this?  (Not to be picky here, but I want to resolve conflicting suggestions in reviews).

Reply:
Sorry, this comment didn't show up because you had already done changes.

Oh, I see they are defined in `SmtEnginePrivate` but essentially implement something in `SmtEngine`. Can we maybe say something along the lines of: "The internal implementation of SmtEngine::setExpressionName()` as described in smt_engine.h"?